resources/windows/msi/main.wxs.erb

```diff
@@ -0,0 +1,781 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'
+  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+  This is how to include wxi files.  We expect this file to define variables
+  like the version numbers and the upgrade UUID.
+  -->
+  <?include $(sys.SOURCEFILEDIR)include/puppet.wxi ?>
+
+  <!--
+  The Product Id should not be changed, the unique identifier enables
+  un-install of the MSI package easily.  More information about supporting
+  major upgrades is available at:
+  http://wix.sourceforge.net/manual-wix3/major_upgrade.htm
+  Name is made of localized product name and version number.
+  -->
+  <Product Id="$(var.ProductCode)" UpgradeCode="$(var.UpgradeCode)"
```

Comment:
Yes - quite a few of them can probably be directly replaced using the ERB operation.